### DISCUSSION TITLE
Disable comparison

### DISCUSSION THREAD

**Discussion by MaMiKoe** [WinMerge/winmerge#1739] [Q&A]
Hello,
I am using winmerge for translastion purposes w/o compare-function.
Yes, a little bit strange, using winmerge just as texteditor. But the advantage is that winmerge has an orientation at the CR+LF and so different length of original and translation doesn´t matter.
The disadvantage is the automatic comparison when loading the textfiles. I do want to compare, it is senseless, so I can not load both text-files when continuing work. I have to load every text into another text-editor, copy the text and paste it into winmerge. Then "save as" two times.
Is there any way to avoid the automatic compare when loading the text-files?

**Comment by sdottaka** [comment]
Unfortunately, it is not possible to disable comparison in WinMerge.

However, if you set the Diff algorithm to none as follows, the lines will not be aligned, and you can change the color of the difference lines to white to make it look like they are not compared.

![image](https://user-images.githubusercontent.com/98126/222293693-ce3c340d-0fc9-4c7f-9bf5-1eeeb856323d.png)

![image](https://user-images.githubusercontent.com/98126/222294012-5b4e649e-9284-402c-94ab-26923dcd19a1.png)

**Reply by MaMiKoe** [reply]
Thanka for your instant reply. I had already done this by my own, but this does not help the the software does the "compare" (e.g. also when doing an "undo"). In the "General"-section I had checked all boxes. Now I have unchecked also as in your screen print wbut w/o changing in th matter of doing the "compare".
Of course I understand that a software which only purpose is to compare two files does this automatically, all the time.

**Comment by gregor202** [comment]
I am currently in a similar situation, where some parts are too different and I want to just edit the files temporarily seperated...
Having a fast Button to activate/deactivate comparison and also the synchronic scrolling would speed up my task a lot.
This feature would be really awesome to have!
